src/handlers/ConfirmStakeIntentHandler.ts

```diff
@@ -0,0 +1,82 @@
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+
+import BigNumber from 'bignumber.js';
+import ContractEntityHandler from './ContractEntityHandler';
+import {
+  MessageDirection,
+  MessageRepository, MessageStatus, MessageType,
+} from '../repositories/MessageRepository';
+import Message from '../models/Message';
+import Logger from '../Logger';
+
+/**
+ * This class handles StakeIntentConfirmed event.
+ */
+export default class ConfirmStakeIntentHandler extends ContractEntityHandler<Message> {
```

Comment:
Name should be StakeIntentConfirmHandler which is consistent with other handlers.